lib/rules/unit-no-unknown/index.js

```diff
@@ -116,8 +117,10 @@ function rule(actual, options) {
 							return;
 						}
 
-						if (/^image-set/i.test(value)) {
-							const imageSet = parsedValue.nodes.find((node) => node.value === 'image-set');
+						if (/image-set/i.test(value)) {
```

Comment:
I think better `/^(?:-webkit-)?image-set$/i`, because in theory we can have `image-set-something()` function (yes, it is not exists in CSS), but strict check always better

Reply:
Yes, let's go with tweaking the RegEx. 

I think we'll be fine with `/^(?:-webkit-)?image-set$/i` as [only the `-webkit` prefix is currently used](https://developer.mozilla.org/en-US/docs/Web/CSS/image-set#Browser_compatibility).

Reply:
Adding `$` at the end does not work here because of what we're matching on. I was able to use `[\s(]` to delimit the end of the function name.